spec/models/king_spec.rb

```diff
@@ -1,5 +1,17 @@
 require 'rails_helper'
 
 RSpec.describe King, type: :model do
-  
+  describe "valid_move?" do
+    it "should check that the new position exists" do
+      Game.create
+      p = [4, -1]
+      expect(King.valid_move?(p)).to eq(false)
```

Comment:
Where is the king piece created? Is it the black or white one? How do you know where the king is placed initially? I am wondering if the test passes..